src/ray/common/ray_config.cc

```diff
@@ -0,0 +1,6 @@
+#include "ray/common/ray_config.h"
```

Comment:
Let's remove this file and put it back inline (I don't think there is a point of having this file).